examples/app/dash/main.py

```diff
@@ -1,3 +1,13 @@
+''' A dashboard that utilizes `Auto MPG`_ and stocks dataset.
+This example demonstrates the use of Bootstrap templates to compile several types
+of charts in one file.
+
+.. note::
+    This chart needs the Pandas package to run.
+
+.. _Auto MPG dataset: https://archive.ics.uci.edu/ml/datasets/auto+mpg
```

Comment:
This link alias here does not match what is used above in single backticks in the docstring. They have to match exactly or Sphinx will not generate a link in the rendered docs